styles/text.scss

```diff
@@ -109,8 +109,25 @@ button:focus-visible {
 
 /* Inline code blocks */
 p > code,
-li > code {
-  @apply border border-gray-40 dark:border-gray-80 text-red-70 rounded-md px-1 mx-1 break-words;
+li > code,
+/* Inline code blocks in docstrings */
+tt.docutils.literal {
+  @apply break-words px-1 rounded;
+  @apply text-green-80 bg-gray-10;
+  @apply dark:text-green-50 dark:bg-gray-90;
+  @apply text-base;
+}
+
+/* Improve `code` links. Don't use border, use underline instead. */
+a:has(tt:only-child),
+a:has(code:only-child) {
+  @apply border-0;
+}
+
+/* Improve `code` links. Don't use border, use underline instead. */
+a > tt:only-child,
+a > code:only-child {
+  @apply underline;
```

Comment:
Inline code links outside of Autofunction have double underline in default text color:
<img width="942" alt="image" src="https://github.com/user-attachments/assets/60c6c7a1-de3e-4500-8ff6-e78a039b8aed" />

Inline code links in Autofunction (function or class description; not parameter description) have a double underline (underline is green, border is default text color)
<img width="905" alt="image" src="https://github.com/user-attachments/assets/c7a58ac1-028e-46c9-98f2-419d06682989" />

Reply:
Great finds! Should be fixed now.